### DISCUSSION TITLE
Does the milvus IP similarity metric has the same definition with faiss' IndexFlatIP?

### DISCUSSION THREAD

**Discussion by wujiangthu** [milvus-io/milvus#3583] [Q&A and General discussion] [answered]
According to milvus' document(https://www.milvus.io/docs/v0.10.0/metric.md), IP instance is actually a normalized dot product, not "real" dot product. And milvus is built on top of faiss, whose IP definition(IndexFlatIP) is dot product.

So is milvus' IP distance dot product, or normalized dot product? is the milvus document something wrong?

**Comment by gujun720** [comment, marked as answer]
@wujiangthu 

It's the same definition. The document you attached is incorrect. We will fix it.

Thanks for pointing out this issue.
